.github/workflows/package-publish.yml

```diff
@@ -121,4 +122,4 @@ jobs:
       uses: pypa/gh-action-pypi-publish@release/v1
       with:
         password: ${{ secrets.PYPI_TEST_PUBLISH_TOKEN }}
-        repository-url: https://test.pypi.org/legacy/
\ No newline at end of file
+        repository-url: https://test.pypi.org/legacy/
```

Comment:
this can be removed, too. I can now set up the project to directly connect to github and not use tokens. Just needed the project to be created in the first instance

Reply:
will leave this to another improvement sweep, we can actually create a reusable action similar to other projects